jenkins/toggle-webhook-processor.py

```diff
@@ -0,0 +1,74 @@
+import boto3
+import sys
+
+
+def main():
+    try:
+        # connect to aws lambda
+        client = boto3.client('lambda')
+    except Exception, e:
+        # if connection failed, raise error
+        raise ValueError(e)
+        sys.exit(0)
+
+    # ensure the number of arguments is correct
+    num_args = len(sys.argv)
+
+    if num_args == 2:
+        # parse toggle argument
+        arg = sys.argv[1]
+
+        try:
+            # find event source mapping for webhook_processor
+            response = client.list_event_source_mappings(
+                FunctionName='webhook_processor'
+            )
+            uuid = response["EventSourceMappings"][0]["UUID"]
+        except:
+            # if none found, lambda does not exist
+            raise ValueError(
+                "No EventSourceMappings found for that lambda function name"
+            )
+            sys.exit(0)
+
+        # call method according to the passed argument
+        if arg == "ON":
+            turn_on(client, uuid)
+        elif arg == "OFF":
+            turn_off(client, uuid)
+        else:
+            raise ValueError("Incorrect number of arguments")
+            sys.exit(0)
+
+
+def turn_on(client, uuid):
+    print "Turning on the event source..."
+
+    # find the previous state, update, and show new state
+    previous_state = get_current_state(client, uuid)
+    client.update_event_source_mapping(UUID=uuid, Enabled=True)
+    current_state = get_current_state(client, uuid)
+
+    print "The state was: " + previous_state
+    print "The state is now: " + current_state
+
+
+def turn_off(client, uuid):
+    print "Turning off the event source..."
+
+    # find the previous state, update, and show new state
+    previous_state = get_current_state(client, uuid)
+    client.update_event_source_mapping(UUID=uuid, Enabled=False)
+    current_state = get_current_state(client, uuid)
+
+    print "The state was: " + previous_state
+    print "The state is now: " + current_state
```

Comment:
You might consider consolidating these two functions, and parameterizing the value for `Enabled`